packages/vitepress-plugin-git-changelog/src/vite/helpers.ts

```diff
@@ -455,12 +455,29 @@ export function findMapAuthorI18n(mappedAuthor: Contributor): Record<string, str
   return undefined
 }
 
+function getAvatarFromGithubNoreplyAddress(email: string | undefined, size: number = 80): string | undefined {
+  if (!email)
+    return undefined
+
+  const match = email.match(/^(?:(?<userId>\d+)\+)?(?<userName>[a-zA-Z\d-]{1,39})@users.noreply.github.com$/)
+  if (!match || !match.groups)
+    return undefined
+
+  const { userName, userId } = match.groups
+  return `https://avatars.githubusercontent.com/${userId ? `u/${userId}` : userName}?size=${size}`
+}
```

Comment:
Can you add unit test for the implementation of this function?

Reply:
Done